modules/core/test/image/testImageWarpAffine.cpp

```diff
@@ -0,0 +1,250 @@
+    \param opt_no_click : Do not need click if set.
+    \param opt_angle : Affine transformation angle of rotation.
+    \param opt_scale_x : Affine transformation scale along x-axis.
+    \param opt_scale_y : Affine transformation scale along y-axis.
+    \param opt_translation_x : Affine transformation translation along x-axis.
+    \param opt_translation_y : Affine transformation translation along y-axis.
+    \return false if the program has to be stopped, true otherwise.
+  */
+  bool getOptions(int argc, const char **argv, std::string &opt_ipath, std::string &opt_ppath,
+                  bool &opt_no_display, bool &opt_no_click, float &opt_angle,
+                  float &opt_scale_x, float &opt_scale_y,
+                  float &opt_translation_x, float &opt_translation_y)
+  {
+    char *opt_ipath_ = NULL;
+    char *opt_ppath_ = NULL;
+
+    vpParseArgv::vpArgvInfo argTable[] =
+    {
+      {"-i", vpParseArgv::ARGV_STRING, (char*) NULL, (char *) &opt_ipath_,
+       "Set image input path. From this path read \"ViSP-images/Klimt/Klimt.pgm\""},
+      {"-p", vpParseArgv::ARGV_STRING, (char*) NULL, (char *) &opt_ppath_,
+       "Path to an image used to test image warping."},
+      {"-c", vpParseArgv::ARGV_CONSTANT, 0, (char *) &opt_no_click,
+       "Disable mouse click."},
+      {"-d", vpParseArgv::ARGV_CONSTANT, 0, (char *) &opt_no_display,
+       "Disable display."},
+      {"-angle", vpParseArgv::ARGV_FLOAT, (char*) NULL, (char *) &opt_angle,
+       "Angle in degrees."},
+      {"-scale_x", vpParseArgv::ARGV_FLOAT, (char*) NULL, (char *) &opt_scale_x,
+       "Scale factor along x."},
+      {"-scale_y", vpParseArgv::ARGV_FLOAT, (char*) NULL, (char *) &opt_scale_y,
+       "Scale factor along y."},
+      {"-trans_x", vpParseArgv::ARGV_FLOAT, (char*) NULL, (char *) &opt_translation_x,
+       "Translation along x."},
+      {"-trans_y", vpParseArgv::ARGV_FLOAT, (char*) NULL, (char *) &opt_translation_y,
+       "Translation along y."},
+      {"-h", vpParseArgv::ARGV_HELP, (char*) NULL, (char *) NULL,
+       "Print the help."},
+      {(char*) NULL, vpParseArgv::ARGV_END, (char*) NULL, (char*) NULL, (char*) NULL}
+    };
+    // Read the command line options
+    if(vpParseArgv::parse(&argc, argv, argTable,
+                          vpParseArgv::ARGV_NO_LEFTOVERS |
+                          vpParseArgv::ARGV_NO_ABBREV |
+                          vpParseArgv::ARGV_NO_DEFAULTS)) {
+      return (false);
+    }
+
+    if (opt_ipath_)
+      opt_ipath = std::string(opt_ipath_);
+    if (opt_ppath_)
+      opt_ppath = std::string(opt_ppath_);
+
+    return true;
+  } 
+}
+
+int main(int argc, const char ** argv) {
+  try {
+    std::string env_ipath;
+    std::string opt_ipath;
+    std::string ipath;
+    std::string opt_ppath;
+    std::string ppath;
+    std::string filename;
+    bool opt_no_display = false;
+    bool opt_no_click = false;
+    // Scale (Sx,Sy), Rotation (angle), Translation (tx,ty)
```

Comment:
`opt_no_display` and `opt_no_click` should be declared as `int` and initialized to `0`